api-documentation.js

```diff
@@ -101,8 +134,9 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
   }
 
   _summaryTemplate() {
-    const { _docsModel, baseUri } = this;
-    return html`<api-summary .amf="${_docsModel}" .baseUri="${baseUri}"></api-summary>`;
+    const { _docsModel } = this;
+
+    return html`<api-summary .amf="${_docsModel}" .baseUri="${this._getBaseUri()}"></api-summary>`;
```

Comment:
Please, don't call functions in a template function. Use getter instead.
And don't change `baseUri` to return either set base URI or selected value. Instead use new getter that returns base URL value in order.